src/ODEs/ODETools/RungeKutta.jl

```diff
@@ -213,12 +214,24 @@ function zero_initial_guess(op::RungeKuttaNonlinearOperator)
   x0
 end
 
-function rhs!(op::RungeKuttaNonlinearOperator, x::AbstractVector)
-  u = x
```

Comment:
I see you are also trying to fix the standard RK method in this PR
It is ok, we can also include it here
But we must check that all tests pass

Reply:
clean version does not edit RungeKutta.jl